src/components/Reservations/MyReservations.js

```diff
@@ -1,7 +1,19 @@
 import React from 'react';
+import { Nav } from 'react-bootstrap';
 
 const MyReservations = () => (
-  <h1>MyReservations page</h1>
+  <div>
+    <h1>MyReservations page</h1>
+    <div className="nav-element">
+      <Nav bg="light" className="main-nav flex-column">
+        <Nav.Link href="/">All Cars</Nav.Link>
+        <Nav.Link href="/Reserve">Reserve</Nav.Link>
+        <Nav.Link href="/Myreservations">My Reservations</Nav.Link>
+        <Nav.Link href="/NewCar">Add a Car</Nav.Link>
+        <Nav.Link>Delete a Car</Nav.Link>
+      </Nav>
```

Comment:
[optional] it will be better if there is separate component for navigation so that it can be reused in every page and encourage dry